modules/openy_features/openy_location/modules/openy_loc_branch/config/install/core.entity_view_display.node.branch.facility.yml

```diff
@@ -0,0 +1,93 @@
+langcode: en
```

Comment:
We should provide update path for all changes in configuration https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Upgrade%20path.md.